### DISCUSSION TITLE
Attachments cannot be downloaded

### DISCUSSION THREAD

**Discussion by wangbabushibie** [dani-garcia/vaultwarden#1684] [Q&A] [answered]
After upgrading openssl and openssh, attachments cannot be downloaded, but they can be uploaded. There is no problem with downloading and uploading before the upgrade.

**Comment by BlackDex** [comment, marked as answer]
Check your DOMAIN setting.
See: `/admin/diagnostics` if it matches.

**Reply by wangbabushibie** [reply]
Solved it, thank you very much

**Reply by wangbabushibie** [reply]
Is it because of the issue of bitwardenrs version update? The DOMAIN was not configured before, but the attachments can be downloaded

**Reply by BlackDex** [reply]
I doubt it. The DOMAIN always needs to be configured, else it would have failed. This is from the start.

**Reply by wangbabushibie** [reply]
I tried it and found that as long as I don’t start the administrator page, I can download attachments without configuring DOMAIN
